fe-nc-news/src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
-import './index.css'
+import './styles/index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
```

Comment:
Looks like you're using StrictMode, which is default when we set up projects using Vite, you don't need to but if you're interested what StrictMode is doing you can take a look at the [React docs](https://react.dev/reference/react/StrictMode)